GeosCore/ucx_mod.F90

```diff
@@ -4139,7 +4139,6 @@ SUBROUTINE NOXCOEFF_INIT( Input_Opt, State_Grid, State_Chm )
 #if defined( MODEL_CESM )
     ENDIF
 
-    CALL MPI_BCAST( State_Chm%NOXCOEFF, nSize, mpi_real8, masterprocid, mpicom, ierr )
```

Comment:
The error catch right below this line should be removed as well.

Reply:
Thanks Lizzie, I also got rid of all the unnecessary variable definitions.